index.html

```diff
@@ -0,0 +1,484 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers | Creative Design Agency</title>
+    <meta name="description" content="Avengers, a Creative Design Agency based out of San Fransisco California, speicalizing in Web Design, UI Design, Icon Design, Branding & Logotype">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <input type="checkbox" id="mobileNavToggle" class="mobileNavToggle">
+    <header class="l-nav">
+      <nav class="mainNav">
+        <ul role="navigation">
+          <li role="menuitem" class="nav-link"><a href="#home">Home.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#services">Services.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#portfolio">Portfolio.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#about">About.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#team">Team.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#contact">Contact.</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page-wrapper" id="home">
+      <label for="mobileNavToggle" class="nav-toggle"></label>
+      <label for="mobileNavToggle" class="trigger"></label>
+      <section class="hero-text-padding hero-image">
+        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="xMidYMid" width="82" height="82" viewBox="0 0 82 82">
+          <defs>
+            <style>
+              .cls-1 {
+                fill: #fd6632;
+              }
+
+              .cls-2 {
+                fill: #fff;
+                stroke: #fff;
+                stroke-linejoin: round;
+                stroke-width: 4px;
+                fill-opacity: 0;
+                fill-rule: evenodd;
+              }
+            </style>
+          </defs>
+          <g>
+            <circle cx="41" cy="41" r="41" class="cls-1"/>
+            <path d="M42.587,23.028 C42.625,23.044 56.680,36.919 56.680,48.673 C56.680,56.228 50.555,62.353 43.000,62.353 C39.242,62.353 35.839,60.838 33.366,58.385 C30.868,55.906 29.320,52.470 29.320,48.673 C29.320,36.919 42.639,23.051 42.587,23.028 Z" class="cls-2"/>
+          </g>
+        </svg>
```

Comment:
Nice use of SVG. Eventually (when we start doing more with JavaScript), this would be a good candidate for the `use` syntax that Jed was talking about in his lecture. However, since the requirement for this project was no JavaScript _and_ it had to work cross-browser, this is a totally valid solution.